core/admin.py

```diff
@@ -18,3 +26,127 @@
 admin.site.register(Organization)
 admin.site.register(Project)
 admin.site.register(DeepDataFetchTracker)
+
+
```

Comment:
Let's define autocomplete fields for foreign fields in the admin panel if there are any. :D